app/javascript/App/bundles/NewsfeedPage/NewsfeedPage.tsx

```diff
@@ -63,11 +65,16 @@ class NewsfeedPage extends React.Component<Props, State> {
   }
 
   componentDidMount() {
+    this.setState({
+      status: STATUSES.LOADING
```

Comment:
Why not setting this in initial state and exec callback here w/o flipping the state?

Reply:
I didn't like that it has a `LOADING` state from the start. But currently, it's working precisely that way. So, I changed it by your suggestion